lib/lambda_ethereum_consensus/p2p/req_resp.ex

```diff
@@ -0,0 +1,89 @@
+defmodule LambdaEthereumConsensus.P2P.ReqRespHandler do
```

Comment:
this is for incoming requests, right?

Reply:
Yes. Do you think `RequestHandler` would be better?

Reply:
IncomingRequestHandler maybe?